CorsixTH/Lua/dialogs/fullscreen/annual_report.lua

```diff
@@ -548,7 +549,7 @@ function UIAnnualReport:draw(canvas, x, y)
       if info.amount > 0 then
         text = "+"
       end
-      self.write_font:draw(canvas, text .. info.amount, x + 220, y + 315, 200, 0)
+      self.write_font:draw(canvas, text .. string.format("%.0f", info.amount), x + 220, y + 315, 200, 0)
```

Comment:
likewise, we can get rid of the previous several lines and include the sign specifier in the format string https://www.cplusplus.com/reference/cstdio/printf/